proto/osmosis/concentrated-liquidity/tickInfo.proto

```diff
@@ -22,7 +22,12 @@ message TickInfo {
     (gogoproto.moretags) = "yaml:\"liquidity_net\"",
     (gogoproto.nullable) = false
   ];
-  repeated cosmos.base.v1beta1.DecCoin fee_growth_outside = 3 [
+  // Total fees accumulated in the opposite direction that the tick was last
+  // crossed. i.e. if the tick was last crossed from left to right, then this is
+  // the total fees accumulated below the tick. If the tick was last crossed
+  // from right to left, then this is the total fees accumulated above the tick.
```

Comment:
```suggestion
  // crossed. i.e. if the current tick is to the right of this tick (meaning its currently a greater price),
  // then this is the total fees accumulated below the tick. 
  // If the current tick is to the left of this tick (meaning its currently at a lower price), 
  // then this is the total fees accumulated above the tick.
```

Reply:
What is the behavior when this tick is the active tick? This variable should just not be used?

Reply:
It depends on what we are calculating. If we are calculating fee growth below and the lower tick is the active tick we use `feeGrowthGlobal` - `FeeGrowthOppositeDirectionOfLastTraversal`. If we are calculating fee growth above and the upper tick is active, we utilize the tick's `FeeGrowthOppositeDirectionOfLastTraversal` directly as the fee growth above. I will add this in the comments as its a very important point, thanks for pointing out :)